src/content/en/updates/2019/08/contact-picker.md

```diff
@@ -119,14 +121,14 @@ In addition, on Android, the Contact Picker requires Android M or later.
 ### Opening the Contact Picker
 
 The entry point to the Contact Picker API is `navigator.contacts.select()`.
-When called, it returns a Promise and shows the Contact Picker, allowing the
+When called, it returns a promise and shows the contact picker, allowing the
```

Comment:
Promise is lower case because it returns an instance, not an a class. Contact picker is not a proper noun.